### PR TITLE
Adds config test for required kernel modules

### PR DESCRIPTION
**Pull request by conorsch**
## Status
Ready for review.

## Description of Changes

These changes add new config tests for VM state, ensuring that the u2mfn kernel module, required for graphical environments in Qubes, was built accurately. Without these tests, the graphical display problems reported in https://github.com/freedomofpress/securedrop-workstation/issues/590 wouild not result in a test suite failure. Now the tests should fail accordingly. 

## Testing

See detailed test plan in https://github.com/freedomofpress/securedrop-debian-packaging/pull/189 to validate the logic changes in the new metapackage. For reviewing this PR, a fully passing `make test` is sufficient. Note that you may see a failing test such as https://github.com/freedomofpress/securedrop-workstation/issues/583 , which is unrelated.

Additionally, it's probably worth confirming that removing the filepath `/usr/lib/modules/4.14.186-grsec-workstation/updates/dkms/u2mfn.ko` from a TemplateVM causes the new test to fail, although be aware that removing that file will break the GUI as described in https://github.com/freedomofpress/securedrop-workstation/issues/590

### If you have made code changes

- [ ] Linter (`make flake8`) passes in the development environment (this box may
      be left unchecked, as `flake8` also runs in CI)

### If you have made changes to the provisioning logic

- [ ] All tests (`make test`) pass in `dom0` of a Qubes install

- [ ] This PR adds/removes files, and includes required updates to the packaging
      logic in `MANIFEST.in` and `rpm-build/SPECS/securedrop-workstation-dom0-config.spec`

### CODE REVIEW

**Comment by emkll** [comment]
Testing for this pr is currently blocked on https://github.com/freedomofpress/securedrop-workstation/issues/606 , the new kernel metapackage being tested here is not yet on apt-test.freedom.press

**Comment by conorsch** [comment]
@emkll You're right that https://github.com/freedomofpress/securedrop-workstation/issues/606 deserves immediate attention (looking into it now) but technically it needn't block here. The ko filepath being tested here has always been present—for each kernel version, and the kernel version has not changed recently—and its absence will result in a non-working GUI. I've amended the plan to point this out. 

Fine to wait on review here, I'd just like to make sure that if the GUI is broken in a way we can observe, we have a corresponding test—otherwise, the config tests say "Everything's fine!" when the AppVMs are unusable.

**Comment by conorsch** [comment]
https://github.com/freedomofpress/securedrop-workstation/pull/606 is re-closed, so ready for testing here.

File: tests/test_vms_exist.py [hunk 1]
```diff
@@ -36,6 +36,12 @@ def _check_kernel(self, vm):
         assert kernel_version.endswith("-grsec-workstation")
         assert kernel_version == EXPECTED_KERNEL_VERSION
 
+        u2mfn_filepath = "/usr/lib/modules/{}/updates/dkms/u2mfn.ko".format(EXPECTED_KERNEL_VERSION)
+        # cmd will raise exception if file not found
+        stdout, stderr = vm.run("sudo test -f {}".format(u2mfn_filepath))
+        assert stdout == ""
```
Comment (emkll) [inline tests/test_vms_exist.py:42]: Looks like adding `.decode("utf-8")` to stdout/stderr resolves the tests for me locally

**Review by emkll — CHANGES_REQUESTED** [review]
Thanks @conorsch for the tests! Looks like the tests are failing for me locally to to string/bytes comparison (see inline).

File: tests/test_vms_exist.py (hunk 1 above)
Comment (conorsch) [reply tests/test_vms_exist.py]: Thanks, you're right! Updated the string checks `""` -> `b""`, now passing for me.

**Review by emkll — APPROVED** [review]
lgtm !

**Pull request merged**